arch/xtensa/src/esp32/Kconfig

```diff
@@ -309,4 +307,58 @@ endif # SERIAL_IFLOWCONTROL || SERIAL_OFLOWCONTROL
 endif # ESP32_UART2
 
 endmenu # UART configuration
+
+menu "SPI configuration"
+	depends on ESP32_SPI
+
+if ESP32_SPI2
+
+config ESP32_SPI2_CSPIN
+	int "SPI2 CS Pin"
```

Comment:
Shouldn't these SPI pins be configured by board logic?  Similar to other boards.

Reply:
Since the pins definition for ESP32 UART and other are already in the Kconfig, I think it is better to keep it this way.